uTensor/ops/MatrixOps.hpp

```diff
@@ -353,7 +535,7 @@ void QuantizedConv(S_TENSOR input, S_TENSOR filter, S_TENSOR output,
   output->resize(c_shape);
 
   //the strides col and row should be decided
-  conv_functor<T1, T2, Toutput>(input, batch, input_rows,
+  quant_conv_functor<T1, T2, Toutput>(input, batch, input_rows,
```

Comment:
In the future we can wrap this in a type selector to make things neat and easy :D

Reply:
No changes needed on your part FYI